<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameObjectTransformationWidgetQt</class>
 <widget class="QWidget" name="GameObjectTransformationWidgetQt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>164</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>426</width>
    <height>164</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Transformation</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow, QWidget, QDialog, QMenuBar::item
{
	background-color: rgb(50, 50, 50);
	color: rgb(255, 255, 255);
}

QHeaderView::section
{
	background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:0.5, stop:0 rgba(50, 50, 50, 255), stop:1 rgba(75, 75, 75, 255));
	padding-left: 4px;
	border: 1px solid rgb(110, 110, 110);
}

QMenuBar::item:selected, QMenu::item:selected
{
    background-color: rgb(120, 120, 120);
	border: 1px solid rgb(200, 200, 200);
}

QPushButton:hover
{
    background-color: rgb(75, 75, 75);
}

QPushButton:pressed
{
    background-color: rgb(100, 100, 100);
}

QLineEdit
{
	border-width: 2px;
	border-style: inset;
	border-color: rgb(60, 60, 60);
	border-radius: 5px;
	padding: 1px;
}

QLineEdit:focus
{
	background-color: rgb(75, 75, 75);
}

QDockWidget::title
{
	background-color: rgb(65, 65, 65);
	padding: 5px;
}

QTabBar::tab 
{
	background-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop:0 rgba(60, 60, 60, 255), stop:0.5 rgba(80, 80, 80, 255), stop:1 rgba(60, 60, 60, 255));
	padding: 5px;
	borde-width: 3px; 
	border-color: rgb(110, 110, 110);
	border-style: inset;
	border-radius: 8px;
	font-size: 11px;
}

QTabBar::tab:selected 
{
	background-color: qlineargradient(spread:pad, x1:1, y1:0.5, x2:0, y2:0.5, stop:0 rgba(60, 60, 60, 255), stop:0.5 rgba(120, 120, 120, 255), stop:1 rgba(60, 60, 60, 255));
	border-style: outset;
    border-bottom-color: rgb(60, 60, 60);
}

QTreeWidget
{
	alternate-background-color: rgb(60, 60, 60);
}

QTreeWidget::branch:has-children
{
	image: url(:/XEngine_Editor/Resources/Images/branch-closegrey.png) 0;
}

QTreeWidget::branch:has-children:open
{
	image: url(:/XEngine_Editor/Resources/Images/branch-opengrey.png) 0;
}

QTreeWidget::branch:has-children:open:selected
{
	image: url(:/XEngine_Editor/Resources/Images/branch-openblue.png) 0;
}

QTreeWidget::branch:has-children:!open:selected
{
	image: url(:/XEngine_Editor/Resources/Images/branch-closeblue.png) 0;
}
</string>
  </property>
  <widget class="QLabel" name="m_LabelPosition">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Position</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_LabelPosX">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_LabelPosY">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>30</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_LabelPosZ">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Z:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_LabelRotX">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_LabelRotation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Rotation</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_LabelRotZ">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Z:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_LabelRotY">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>80</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_LabelSclZ">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Z:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_LabelSclY">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>130</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_LabelSclX">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_LabelScale">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_PosX">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>28</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>-50000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>50000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_PosY">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>28</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>-50000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>50000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_PosZ">
   <property name="geometry">
    <rect>
     <x>308</x>
     <y>28</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>-50000.000000000000000</double>
   </property>
   <property name="maximum">
    <double>50000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_RotX">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>78</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-361.000000000000000</double>
   </property>
   <property name="maximum">
    <double>361.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_RotY">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>78</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-361.000000000000000</double>
   </property>
   <property name="maximum">
    <double>361.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_RotZ">
   <property name="geometry">
    <rect>
     <x>308</x>
     <y>78</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>-361.000000000000000</double>
   </property>
   <property name="maximum">
    <double>361.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_SclX">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>128</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.001000000000000</double>
   </property>
   <property name="maximum">
    <double>50000.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_SclY">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>128</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.001000000000000</double>
   </property>
   <property name="maximum">
    <double>50000.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="m_SclZ">
   <property name="geometry">
    <rect>
     <x>308</x>
     <y>128</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.001000000000000</double>
   </property>
   <property name="maximum">
    <double>50000.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../xengine_editor.qrc"/>
 </resources>
 <connections/>
</ui>
